### PR TITLE
CreateSite Dialog Feedback and Issues

### PR DESCRIPTION
**Pull request by aachavarria**
Fixing overlay issues on MUI and Angular modal

### CODE REVIEW

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
@yureii00 @rart please don't push code after approval. I'll await a new approval because there was a push after the initial approval.

**Review by rart — APPROVED** [review]

**Pull request merged**
